Tests/TuistLoaderTests/SwiftPackageManager/PackageInfoMapperTests.swift

```diff
@@ -3304,6 +3304,251 @@ final class PackageInfoMapperTests: TuistUnitTestCase {
             )
         )
     }
+
+    func testMap_whenSwiftPackageHavingTestTarget() throws {
+        // Given
+        let basePath = try temporaryPath()
+        let sourcesPath = basePath.appending(try RelativePath(validating: "Package/Sources/Target"))
```

Comment:
```suggestion
        let sourcesPath = basePath.appending(components: "Package", "Sources", "Target")
```

nit: this is imho easier to read and does the same

Reply:
I agree.